acquisitions/src/main/resources/thunderjet/mod-orders/orders-junit.feature

```diff
@@ -32,6 +32,7 @@ Feature: mod-orders integration tests
       | 'orders.item.unopen'                   |
       | 'invoice.all'                          |
       | 'audit.all'                            |
+      | 'inventory-storage.holdings.item.post' |
```

Comment:
Should it be in module permissions instead?

Reply:
I removed this permission from here and added it to the patch method from mod-orders.